packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -169,11 +179,11 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
 
   self.result = result;
   _arguments = call.arguments;
+  BOOL usePHAsset = [[_arguments objectForKey:@"requestFullMetadata"] boolValue];
 
   if ([@"pickImage" isEqualToString:call.method]) {
     int imageSource = [call.arguments[@"source"] intValue];
-
-    if (imageSource == SOURCE_GALLERY) {  // Capture is not possible with PHPicker
+    if (usePHAsset && imageSource == SOURCE_GALLERY) {  // Capture is not possible with PHPicker
```

Comment:
I'm confused by this change. Why is forcing `UIImagePicker` desirable, given that `PHPicker` also shouldn't require permission prompting? Isn't the permission issue specifically related to accessing metadata, not the top-level API used?

Reply:
You're right, for `PHPicker` it's enough to prevent from calling util method `getAssetFromImagePickerInfo`. I'll fix that part